Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -98,22 +132,31 @@ import BraintreeCore
     ///   See https://developer.paypal.com/docs/api/reference/currency-codes/ for a list of supported currency codes.
     ///   - requestBillingAgreement: Optional: If set to `true`, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement
     ///   during checkout. Defaults to `false`.
+    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    ///   - enablePayPalAppSwitch: Optional: Used to determine if the customer will use the PayPal app switch flow. Defaults to `false`.
```

Comment:
I think we only want the `enablePayPalAppSwitch` property on the app switch init, this will ensure merchants pass us the correct properties if opting in for this flow. I believe we do the same in `BTPayPalVaultRequest`. What do you think?

Reply:
ahhh gotcha, i see that in the `BTPayPalVaultRequest` we pass in both app-switch properties: https://github.com/braintree/braintree_ios/blob/2f3dd67f32218888f9738d8629a1a89897956eaf/Sources/BraintreePayPal/BTPayPalVaultRequest.swift#L37-L41

just wanted to confirm, did you mean we can exclude the `userAuthenticationEmail` from the app-switch init on `BTPayPalCheckoutRequest` 👀

Reply:
Sorry I should have been more specific. We _do_ want it on the app switch convenience init. In the init linked above you will see we set `self.enablePayPalAppSwitch = enablePayPalAppSwitch`. But on the other init below that here: https://github.com/braintree/braintree_ios/blob/2f3dd67f32218888f9738d8629a1a89897956eaf/Sources/BraintreePayPal/BTPayPalVaultRequest.swift#L52-L57 we do not have the `enablePayPalAppSwitch` property included. This allows us to ensure that if `enablePayPalAppSwitch = true` the merchant is also passing a currently required `userAuthenticationEmail`.

Let me know if that doesn't make sense though and we can always grab some time to pair on it!